packages/lib/theme.ts

```diff
@@ -97,7 +97,49 @@ const globalStyle = (() => {
 	};
 })();
 
-export function extraStyles(theme: Theme) {
+export const derivedColors = (theme: Theme) => {
```

Comment:
Since `scrollbarThumbColor` is used in `renderer/`, it needs to be available on mobile. As such, derived colors were refactored into a new `derivedColors` function that can be used in `app-mobile`'s `global-style.ts`.